molecule/playbooks/prepare.yml

```diff
@@ -124,13 +134,14 @@
         exclusive: False
       when: (ansible_distribution == "CentOS") and (lookup('env', 'DESTROY_ENV') == "no")
 
-    - name: Setup SSH keys Oracle Linux
+    - name: Setup SSH keys for Oracle Linux or Amazon
       authorized_key:
         user: ec2-user
         key: "{{ lookup('file', 'public_keys') }}"
         state: present
         exclusive: False
-      when: (ansible_distribution == "OracleLinux") and (lookup('env', 'DESTROY_ENV') == "no")
+      when: (ansible_distribution == "OracleLinux") and (lookup('env', 'DESTROY_ENV') == "no") or
+            (ansible_distribution == "amazon") and (lookup('env', 'DESTROY_ENV') == "no")
```

Comment:
Is it lowercase or uppercase (as in "setup epel 7 repo for Amazon Linux" 2 task)

Reply:
Uppercase , fixed